src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -146,9 +152,9 @@ export default compose(
     withLocalize,
     withWindowDimensions,
     withOnyx({
-        isCloseAccoutModalOpen: {
-            key: ONYXKEYS.IS_CLOSE_ACCOUNT_MODAL_OPEN,
-            initWithStoredValues: false,
+        closeAccountData: {
```

Comment:
NAB, I would view `props.closeAccount.error` the same as `props.closeAccountData.error` and the "data" doesn't tell me more - everything coming from Onyx is "data".

Reply:
>However, I don't really like having an onyx key (ONYXKEYS.CLOSE_ACCOUNT) stored in a variable with a different name (closeAccountData)

Haha well this kind of convinces me that we should change that and update the key or variable at least.

>closeAccount sounds like a function

Kinda see it, but also Onyx does not return functions it returns data so unsure why this would be confusing or bad.

Could we just store this in the `account` object?

Reply:
> Kinda see it, but also Onyx does not return functions it returns data so unsure why this would be confusing or bad.

True, it's not actually too annoying to look at the prop type 🤷 I think it's easiest to update the prop to `closeAccount` :D